socialhome/streams/app/components/Stream.vue

```diff
@@ -52,6 +55,19 @@ export default Vue.component("stream", {
             },
         }
     },
+    computed: {
+        stampedElement() {
+            if (this.$store.state.streamName.match(/followed/)) {
```

Comment:
You must use `starts with` kind of check OR do a split and take the first element. For example currently tag streams have the tag name slug in the stream name, so #followed would get the followed stamped element. In a way this is bad and unnecessary, but it's the way it is. And the "spec" for stream names is basically `<stream_type>__<id or other identifier>`. The first two underscores end the stream type definition which should be used.

Reply:
`/^followed/` might be enough, then.